charts/cluster-security/templates/bindings.yaml

```diff
@@ -1,30 +1,35 @@
-# This role binding allows all pods in the 'default' and 'apps' namespace to use the baseline PSP.
-apiVersion: rbac.authorization.k8s.io/v1
+{{- range .Values.baselineNamespaces }}
 kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: psp-baseline-rolebinding
-  namespace: apps
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
-  name: psp-baseline
+  namespace: {{ . }}
 subjects:
 - kind: Group
   name: system:authenticated
-  namespace: apps
+  namespace: {{ . }}
+  apiGroup: rbac.authorization.k8s.io
+roleRef:
+  kind: ClusterRole
+  name: psp-baseline-clusterrole
   apiGroup: rbac.authorization.k8s.io
 ---
-apiVersion: rbac.authorization.k8s.io/v1
+{{ end }}
+---
+{{- range .Values.privilegedNamespaces }}
 kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: psp-baseline-rolebinding
-  namespace: default
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
-  name: psp-baseline
+  name: psp-privileged-rolebinding
+  namespace: {{ . }}
 subjects:
 - kind: Group
   name: system:authenticated
```

Comment:
Sorry didn't spot this earlier - it's best not to use `system:authenticated` here. The [docs](https://kubernetes.io/docs/concepts/security/pod-security-policy/#recommended-practice) recommend authorising just all service accounts in the namespace. (I'd also include cluster admins so that we don't break things like `kubectl run`, which we'll likely need for troubleshooting at least in the short term.)